includes/email-templates.php

```diff
@@ -55,7 +55,7 @@
 	'billable_invoice' => array(
 		'subject' => __( 'Invoice for order #: !!order_code!!', 'paid-memberships-pro' ),
         'description' => __( 'Billable Invoice', 'paid-membershps-pro' ),
-		'body' => __( '<p>Thank you for your membership to !!sitename!!. Below is your invoice for order #: !!order_code!!</p>
+		'body' => __( '<p>Thank you for your membership to !!membership_level_name!! at !!sitename!!. Below is your invoice for order #: !!order_code!!</p>
```

Comment:
Other email templates, such as all the checkout templates, do not include the level name that was purchased. Reverting this change to stay consistent with those other email template defaults.
```suggestion
		'body' => __( '<p>Thank you for your membership to !!sitename!!. Below is your invoice for order #: !!order_code!!</p>
```